include/rocksdb/compaction_filter.h

```diff
@@ -54,6 +55,10 @@ class CompactionFilter {
     bool is_manual_compaction;
     // Which column family this compaction is for.
     uint32_t column_family_id;
+
+    Slice start_key;
+    Slice end_key;
+    bool is_end_key_inclusive;
```

Comment:
Seems the `is_end_key_inclusive` never use?

Reply:
this is mean to used by the client compaction filter.